### DISCUSSION TITLE
HelloGitHub 徽章计划

### DISCUSSION THREAD

**Discussion by 521xueweihan** [dukeluo/wai#110] [General]
🎉 恭喜！您的[项目](https://hellogithub.com/repository/0b99a938e238439dbf6bc0660c5bcdb0)已被 HelloGitHub 社区收录和推荐，现邀请您加入 HelloGitHub 徽章计划。加入后，您将获得以下特权：

- 社区认可: 徽章代表你的项目已通过 HelloGitHub 社区的严格筛选，并获得推荐。
- 提高曝光: 佩戴徽章后，你的项目将获得更多推荐流量，吸引更多用户和贡献者。
- 增强互动: 用户可通过徽章快速了解项目，并与之互动（点赞、评论、收藏）。
- 收集反馈: 获取广大用户的真实反馈，持续优化你的项目。
- 特殊标识: 认证后，你的评论将有显著标识，并获得置顶优先权。

📌 [立即点击](https://hellogithub.com/repository/0b99a938e238439dbf6bc0660c5bcdb0/embed) 完成徽章佩戴，加入 HelloGitHub 徽章计划，让你的开源项目绽放更耀眼的光彩。

---

**HelloGitHub**，一个专注于发现、分享、推广开源项目的社区。自 2016 年创立以来，从每月一更的月刊发展到今天，已经成为一个拥有超过 **1 万** 用户的充满活力的开源社区。我们的足迹遍布多个内容平台，并获得了全网 **50 万** 粉丝的信赖与支持。

<img width="775" alt="image" src="https://github.com/dukeluo/wai/assets/8255800/96b0aba2-0831-4270-9201-465d820659a0">

**Comment by 521xueweihan** [comment]
@dukeluo  感谢您的认领和支持，认领已通过并全站置顶您的项目（1 天）：

![image](https://github.com/PointRider/open-java/assets/8255800/d18d77dc-780f-4bf0-9718-ec440dc84589)

**Reply by dukeluo** [reply]
谢谢！
